### PR TITLE
f-loyalty@v1.0.0 - loading component

### PR DESCRIPTION
**Pull request by matthewhardern**
PR adds loading component for loyalty cards component. Also added unit test for the component.

### CODE REVIEW

**Review by minrwhite — APPROVED** [review]

File: packages/components/organisms/f-loyalty/src/components/Loading.vue [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    margin: spacing();
+    width: $loyalty-loadingCardWidthDesktop;
+    height: $loyalty-loadingCardHeight;
+    padding: spacing(x2);
+    border-radius: $border-radius;
+    box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.03),
+    0 3px 1px -2px rgba(0, 0, 0, 0.07),
+    0 1px 5px 0 rgba(0, 0, 0, 0.06);
+
+    /** START placeholder graphic details */
```
Comment (zlatinivanov-je) [inline packages/components/organisms/f-loyalty/src/components/Loading.vue:66]: Is this a repeating pattern? Maybe consider at some point creating a generic state, available to others

File: packages/components/organisms/f-loyalty/src/components/Loading.vue [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            <div
+                v-for="n in cardCount"
+                :key="`loadingStampCard-${n}`"
+                :class="$style['c-loyalty-loadingCardPlaceholder']" />
+        </div>
+    </div>
+</template>
+
+<script>
+const NARROW_BREAKPOINT_WIDTH = 600;
```
Comment (zlatinivanov-je) [inline packages/components/organisms/f-loyalty/src/components/Loading.vue:14]: I have the habit of putting those on config files and referring to them in my tests

**Review by zlatinivanov-je — APPROVED** [review]

File: packages/components/organisms/f-loyalty/src/components/Loading.vue (hunk 1 above)
Comment (matthewhardern) [reply packages/components/organisms/f-loyalty/src/components/Loading.vue:66]: yes think this is a good idea

File: packages/components/organisms/f-loyalty/src/components/Loading.vue (hunk 2 above)
Comment (matthewhardern) [reply packages/components/organisms/f-loyalty/src/components/Loading.vue:14]: i will probably end up doing this when all the bits are pulled together, but good point

**Pull request merged**